Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -5590,5 +5590,74 @@ def test_failure(self):
         return self.runStep()
 
 
+class TestValidateSquashed(BuildStepMixinAdditions, unittest.TestCase):
+    def setUp(self):
+        self.longMessage = True
+        return self.setUpBuildStep()
+
+    def tearDown(self):
+        return self.tearDownBuildStep()
+
+    def configureStep(self):
```

Comment:
Creating this separate function probably doesn't add much value in this case. It is still one line call `self.configureStep()` instead of more explicit `self.setupStep(ValidateSquashed())`.